### DISCUSSION TITLE
Colored IP adress in a Talbe

### DISCUSSION THREAD

**Discussion by samsja** [Textualize/rich#2025] [Q&A] [answered]
Hey, is there a way to correctly color an IP adress within a table ?

right now it is not rendered as nicely as a rich print

```python
from rich.table import Table
from rich import print


if __name__ == '__main__':
    ip = "192.168.1.1:8080"

    table = Table(title=None, box=None)
    table.add_column("", justify="right", style="cyan")
    table.add_row(ip)

    print(table)
    print(ip)
```

![Screenshot from 2022-03-04 15-33-56](https://user-images.githubusercontent.com/55492238/156782290-0dda9ca7-9282-4134-9695-ae0442348510.png)

**Comment by willmcgugan** [comment, marked as answer]
You can set `highlight=True` on the table.

**Reply by samsja** [reply]
Thank you !!
